src/themes/animations.js

```diff
@@ -1,22 +1,18 @@
 'use strict';
 
 export default {
-    toggle: (props) => {
-        return {
-            animation: { rotateZ: props.node.toggled ? 90 : 0 },
+    toggle: ({node: toggled}) => ({
```

Comment:
hi @MichaelDeBoey . I think here is a little mistake. The correct code should be:
```
 export default {
    toggle: ({node}) => ({
        animation: {rotateZ: node.toggled ? 90 : 0},
        ...
```